cengage/react-magma
# Dropdown Expandable - QA review tweaks from Laura and Orion

Issue: # [1014](https://github.com/cengage/react-magma/issues/1014)

## What I did
- CSS tweaks to component including outline and padding adjustments.
- Added doc example for 'defaultIndex', fixed missing references.
- Removed unnecessary prop inclusions.

## Checklist 
- [N/A] I have added tests that prove my fix is effective or that my feature works

## How to test
- Make sure all Codesandbox examples load properly.
- Verify padding and outline fixes are appropriately tuned.
- Confirm the 'iconPosition' doesn't change layout when set to 'left' or 'none' on 'DropdownExpandableMenuGroup'.
- Confirm the 'disabled' prop only works for 'DropdownExpandableMenuItem'.
- Rejoice
